upgrade/upgrade-scripts/execute

```diff
@@ -459,6 +468,50 @@ else
 			"for '$ROOTFS_CONTAINER'"
 fi
 
+# In AWS environments, changing instance type could result in generation of
```

Comment:
Please make sure to run this through shfmt

Reply:
Yes, all the comments from shfmt have been incorporated.